File: financial/Models/CloseOrderDTO.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Models
{
    public class CloseOrderDTO
    {
        public string PostalCode { get; set; }
        public int EstablishmentId { get; set; }
        public string Id { get; set; }

        public DeliveryRegion DeliveryRegion { get; set; }
        public Address Address { get; set; }
        public ApplicationUserDTO ApplicationUserDTO { get; set; }
        public Establishment Establishment { get; set; }
    }
}

File: android/app/src/main/java/com/br/app/deliverydelicatessen/model/CloseOrderDTO.kt
package com.br.app.deliverydelicatessen.model

data class CloseOrderDTO(
    var postalCode: String = "",
    var establishmentId: Int = 0,
    var deliveryRegion: DeliveryRegion? = null,
    var address: Address? = null,
    var applicationUserDTO: ApplicationUserDTO? = null,
    var establishment: Establishment? = null,
)